### DISCUSSION TITLE
Change to page '/how-to-contribute/index'

### DISCUSSION THREAD

**Discussion by Helen326** [nhsbsa/nhsbsa-digital-playbook#294] [General Playbook Ideas]
Some ideas on improvements to the updating the playbook guidance (mainly for those unfamiliar with the process):
•	Include who to get in touch with in case of questions
•	Including how to update the playbook for use on Windows
•	Index guidance/front matter – could we add more details to include more information of the collections etc?
•	Useful extensions within Visual Studio Code could be good to include
•	Details around the working group and it’s purpose. What is the process for someone who needs advice/who can they go to? Would it be worth holding an open forum sometimes to ask questions?
•	Detailing the process of raising discussions etc – when would you do this, who picks it up and does someone get back to the individual who raised it?
•	Where do you run npm install etc? in the terminal Quick Start. Guidance to be accessible to people who haven’t used Git or Markdown before. Also a list of what you need to install beforehand, e.g. Node etc.
•	Contributing Git guidance – how to ‘Fork the project; etc. examples of unit tests etc might be useful to include
•	Mention linking issues to the pull requests and this process

**Comment by isobelthompson** [comment]
I would second these points. I would also add to the second-last point about giving more detail on forking/cloning the repo (which should we be doing?) and setting it up in VSCode. I had used github desktop but never VSCode and find there is a lot of conflicting or vague advice online in general, so something very comprehensive and specific for this project would be helpful.

**Comment by nhsbsa-sthil** [comment]
I'll defer to @yantantether etc... around advice on improving the technical guidance etc... but just wanted to say that it's great to see other disciplines other than Developers, Designers etc... who feel confident enough to be able to make the technical changes to the playbook.

I don't think it was ever an expectation that other disciplines would make the changes but anything to make this more inclusive can only be a good thing! 😄
